spec/synchronization.lua

```diff
@@ -0,0 +1,78 @@
+describe("Synchronization", function()
```

Comment:
this file needs to be named `sychronization_spec.lua`. Without the `_spec` our Makefile will not run the test in this file.